content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -0,0 +1,430 @@
+
+When using the following schema:
+
+```prisma
+datasource db {
+  provider = "postgresql"
+}
+
+model User {
+  id       String @id
+  settings Json
+}
+```
+
+Prisma 4 and lower:
+
+```js
+prisma.user.findMany({
+  where: {
+    settings: {
+      path: 'someSetting',
+      equals: someValue,
+    },
+  },
+})
+```
+
+Prisma 5:
+
+```js highlight=4;edit
+prisma.user.findMany({
+  where: {
+    settings: {
+      path: ['someSetting'],
+      equals: someValue,
+    },
+  },
+})
+```
+
+<Admonition type="info">
+
+Note: This change only affects PostgreSQL databases. MySQL databases are not affected.
+
+</Admonition>
+
+#### Composite and Scalar lists
+
+Inputs for operations on lists of [Composite](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#defining-composite-types) and [Scalar](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#scalar-fields) types now only accept array values. The following examples all use the following schema:
+
+```prisma
+model Post {
+  id String @id @default(uuid())
+  commentsList Comment[]
+  tags String[]
+}
+
+type Comment {
+  text String
+}
+```
+
+##### Composite list filtering
+
+[When filtering records based on a composite list field](https://www.prisma.io/docs/concepts/components/prisma-client/composite-types#finding-records-that-contain-composite-types-with-find-and-findmany) you will only be able to supply array values.
+
+Prisma 4 and lower:
+
+```js
+prisma.post.findMany({
+  commentsList: { equals: { text: 'hello' } },
+})
+```
+
+Prisma 5:
+
+```js highlight=2-4;normal
+prisma.post.findMany({
+  commentsList: {
+    equals: [{ text: 'hello' }],
+  },
+})
+```
+
+##### Composite list input
+
+If you exclude `equals`, you still must supply an array value for composite list fields.
+
+Prisma 4 and lower:
+
+```js
+prisma.post.findMany({
+  commentsList: { text: 'hello' },
+})
+```
+
+Prisma 5:
+
+```js highlight=2;normal
+prisma.post.findMany({
+  commentsList: [{ text: 'hello' }],
+})
+```
+
+##### Scalar lists
+
+Scalar list values must be arrays in all operations.
+
+Prisma 4 and lower:
+
+```js
+prisma.post.create({
+  data: { tags: 'databases' },
+})
+
+prisma.post.findMany({
+  where: { tags: 'databases' },
+})
+```
+
+Prisma 5:
+
+```js highlight=3,9;edit
+prisma.post.create({
+  data: {
+    tags: ['databases'],
+  },
+})
+
+prisma.post.findMany({
+  where: {
+    tags: ['databases'],
+  },
+})
+```
+
+### Prisma Client embedded SQLite version updated
+
+With Prisma version 5.0.0, we have upgraded the embedded version of SQLite from `3.35.4` to `3.41.2`. We did not see any breaking changes and don't anticipate any changes needed in user projects, but if you are using SQLite, make sure to check [the SQLite changelog](https://www.sqlite.org/changes.html)
+
+### Removal of <inlinecode>jsonProtocol</inlinecode> preview feature
```

Comment:
On first read, this sounds like we're removing a feature, vs. moving it from preview to GA, or removing the feature flag. Could we make it sound less destructive?

Reply:
(Blog post and release notes have some ideas for positive spin)